README.md

```diff
@@ -10,135 +9,438 @@ Current supported features are:
+Next, edit the Apache configuration to load the application. Here's a basic
+example of what it looks like:
 
-	WSGIScriptAlias /supysonic /path/to/supysonic/cgi-bin/supysonic.wsgi
-	<Directory /path/to/supysonic/cgi-bin>
-		WSGIApplicationGroup %{GLOBAL}
-		WSGIPassAuthorization On
-		Order deny,allow
-		Allow from all
-	</Directory>
+    WSGIScriptAlias /supysonic /path/to/supysonic/cgi-bin/supysonic.wsgi
+    <Directory /path/to/supysonic/cgi-bin>
+        WSGIApplicationGroup %{GLOBAL}
+        WSGIPassAuthorization On
+        Order deny,allow
+        Allow from all
+    </Directory>
 
-You might also need to run Apache using the system default locale, as the one it uses might cause problems while
-scanning the library. To do so, edit the `/etc/apache2/envvars` file, comment the line `export LANG=C` and
-uncomment the `. /etc/default/locale` line. Then you can restart Apache.
+You might also need to run Apache using the system default locale, as the one
+it uses might cause problems while scanning the library. To do so, edit the
+`/etc/apache2/envvars` file, comment the line `export LANG=C` and uncomment
+the `. /etc/default/locale` line. Then you can restart Apache:
 
-	service apache2 restart
+    $ service apache2 restart
 
 With that kind of configuration, the server address will look like *http://server/supysonic/*
 
 ### Other options
 
-If you use another HTTP server, such as *nginx* or *lighttpd*, or prefer to use FastCGI or CGI over WSGI,
-FastCGI and CGI scripts are also providedin the `cgi-bin` folder, respectively as `supysonic.fcgi` and `supysonic.cgi`.
-As with WSGI, you might need to edit those file to suit your system configuration.
+If you use another HTTP server, such as *nginx* or *lighttpd*, or prefer to
+use FastCGI or CGI over WSGI, FastCGI and CGI scripts are also provided in the
+`cgi-bin` folder, respectively as `supysonic.fcgi` and `supysonic.cgi`. As with
+WSGI, you might need to edit those file to suit your system configuration.
+
+Here are some quick docs on how to configure your server for [FastCGI][] or [CGI][].
+
+[fastcgi]: http://flask.pocoo.org/docs/deploying/fastcgi/
+[cgi]: http://flask.pocoo.org/docs/deploying/cgi/
+
+## Transcoding
+
+Transcoding is the process of converting from one audio format to another. This
+allows for streaming of formats that wouldn't be streamable otherwise, or
+reducing the quality of an audio file to allow a decent streaming for clients
+with limited bandwidth, such as the ones running on a mobile connection.
+
+Supysonic's transcoding is achieved through the use of third-party command-line
+programs. Supysonic isn't bundled with such programs, and you are left to
+choose which one you want to use.
+
+If the client you are using is not specifying a transcoding format, you will
+need to do this on the web interface, otherwise supysonic will not transcode
+for that client. You can also specify the maximum bitrate to use there.
```

Comment:
The wording is strange here. Something along the lines of *"If you want to use transcoding but your client doesn't allow you to do so, you can force Supysonic to transcode for that client by going on the web interface"* would be better I think.